lib/LedgerSMB/Admin/Command/destroy.pm

```diff
@@ -0,0 +1,177 @@
+
+package LedgerSMB::Admin::Command::destroy;
+
+=head1 NAME
+
+LedgerSMB::Admin::Command::destroy - ledgersmb-admin 'destroy' command
+
+=cut
+
+use strict;
+use warnings;
+
+use Syntax::Keyword::Try;
+
+use LedgerSMB::Admin::Command;
+use LedgerSMB::Database;
+
+use Moose;
+extends 'LedgerSMB::Admin::Command';
+use namespace::autoclean;
+
+my $logger;
```

Comment:
@ylavoie why not simply declare the logger internal to the `run()` method?

Reply:
It is used in other functions